src/foam/core/AbstractFObject.js

```diff
@@ -237,6 +237,7 @@ foam.CLASS({
             // nop - ignore - only copy common properties.
           }
         }
+        this.__frozen__ = false;
```

Comment:
Why are you doing this? Once an object is frozen it needs to be frozen forever.

Reply:
this was just testing.  Needs to be removed.